integrations/snyk/.port/resources/port-app-config.yaml

```diff
@@ -30,6 +30,8 @@ resources:
             language: .issueData.language // .issueType
             publicationTime: .issueData.publicationTime
             isPatched: .isPatched
+          relations:
+            project: '.links.paths | split("/") | .[8]'
```

Comment:
Are you certain it will always be at that position? no other way to get this data?

Reply:
Yes, it will be, as the data is obtained from a [specific API version](https://apidocs.snyk.io/?_gl=1*1se7g7g*_ga*MTQyMDcwOTQ2My4xNzE3MDgxNDE5*_ga_X9SH3KP7B4*MTcxODAzMTUxNy4zLjEuMTcxODAzMTUzNC40My4wLjA.&version=2023-08-21#get-/openapi/-version-) (2023-08-21). 

Being that they follow the OpenAPI spec on versioning, we have assurance that unless the version is retired then it will continue working as it.